Source/ChocolateyGui/Properties/Resources.nb.resx

```diff
@@ -0,0 +1,553 @@
+    <value>Galleri</value>
+  </data>
+  <data name="PackageView_LastUpdate" xml:space="preserve">
+    <value>Sist oppdatert</value>
+  </data>
+  <data name="PackageView_License" xml:space="preserve">
+    <value>Lisens</value>
+  </data>
+  <data name="PackageView_Maintainers" xml:space="preserve">
+    <value> | Vedlikeholdere: </value>
+  </data>
+  <data name="PackageView_PackageID" xml:space="preserve">
+    <value>Pakke ID</value>
+  </data>
+  <data name="PackageView_PackageSize" xml:space="preserve">
+    <value>Pakke Størrelse</value>
+  </data>
+  <data name="PackageView_ProjectSite" xml:space="preserve">
+    <value>Prosjektside</value>
+  </data>
+  <data name="PackageView_ReleaseNotes" xml:space="preserve">
+    <value>Produktmerknader</value>
+  </data>
+  <data name="PackageView_ReportAbuse" xml:space="preserve">
+    <value>Rapporter misbruk</value>
+  </data>
+  <data name="PackageView_Summary" xml:space="preserve">
+    <value>Sammendrag</value>
+  </data>
+  <data name="PackageView_TotalDownloads" xml:space="preserve">
+    <value>Totalt Nedlastet</value>
+  </data>
+  <data name="RemoteSourceView_Authors" xml:space="preserve">
+    <value>Forfattere:</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGoBackAPage" xml:space="preserve">
+    <value>Gå tilbake en side</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGoForwardAPage" xml:space="preserve">
+    <value>Neste</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGotoFirstPage" xml:space="preserve">
+    <value>Gå til første side</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGotoLastPage" xml:space="preserve">
+    <value>Siste</value>
+  </data>
+  <data name="RemoteSourceView_ButtonRefreshPkgs" xml:space="preserve">
+    <value>Oppdater Pakkeliste</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxAllVersions" xml:space="preserve">
+    <value>Alle Versjoner</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxIncludePrerelease" xml:space="preserve">
+    <value>Inkluder Forhåndsversjoner</value>
+  </data>
+  <data name="RemoteSourceView_Downloads" xml:space="preserve">
+    <value>nedlastinger</value>
+  </data>
+  <data name="RemoteSourceView_SearchBoxText" xml:space="preserve">
+    <value>Søk:</value>
+  </data>
+  <data name="Resources_ThisPC" xml:space="preserve">
+    <value>Denne Datamaskinen</value>
+    <comment>Potentially change to Denne datamaskinen (this computer)</comment>
+  </data>
+  <data name="SettingsView_About" xml:space="preserve">
+    <value>Om</value>
+  </data>
+  <data name="SettingsView_ButtonCancel" xml:space="preserve">
+    <value>Avbryt</value>
+  </data>
+  <data name="SettingsView_ButtonNew" xml:space="preserve">
+    <value>Ny</value>
+  </data>
+  <data name="SettingsView_ButtonSave" xml:space="preserve">
+    <value>Lagre</value>
+  </data>
+  <data name="SettingsView_ChocoGui" xml:space="preserve">
+    <value>Chocolatey Gui</value>

```

Comment:
yup, stemmer det.
Egentlig så kunne jeg jo ha fjernet denne, ettersom den vanlige `Resource.resx` filen hadde blitt brukt for denne strengen istedenfor.